examples/SESAR/test/iSamplesEOI00002HBasic.json

```diff
@@ -3,20 +3,20 @@
     "@id": "https://data.isamples.org/digitalsample/igsn/EOI00002H",
     "label": "J730-GTHFS-16",
     "sampleidentifier": "igsn:EOI00002H",
-    "description": "",
+    "description": "Not Provided",
```

Comment:
Given that one of the sample files had this, we needed to be consistent here and use this in lieu of the empty string.